### PR TITLE
Sieve of Eratosthenes and Linear Search

### PR DESCRIPTION
**Pull request by shaswatsingh19**
## Have you read the Contribution Guidelines?

YES

## Description

Implemented Sieve of Eratosthenes in python to find all prime in a particular range

## Checklist
- [tick] I've read the contribution guidelines.
- [tick] I've checked the issue list before deciding what to submit.
- [tick] I've edited the README.md and link to my code.
 
 
## Related Issues (Mention Issue no. like #xxx)

Fixes Issue #52 #39

### CODE REVIEW

**Comment by sukritishah15** [comment]
@shaswatsingh19 
Make use of multiline comments.
In my original repo. https://github.com/sukritishah15/DS-Algo-Point/blob/master/Python/README.md
already has 4 links.
Please update your repo.

**Comment by shaswatsingh19** [comment]
sorry I don't understand please can you describe 'already has 4 links' means

**Comment by sukritishah15** [comment]
> sorry I don't understand please can you describe 'already has 4 links' means

Just take care of multiline comments @shaswatsingh19 
I'll take care of the rest. 👍 
Thank you!

**Comment by shaswatsingh19** [comment]
Yeah I have changed

File: Python/Sieve_of_Eratosthenes.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+# Checking if a particular value is prime or not
+check = int(input('\nWrite a number to check whether it is prime or not until '+str(n)+ '\n'))
+if primes[check] == 1:
+    print(check,'is a prime number')
+else:
+    print(check,'is not a prime number')
+
+
+
+# I/O ---->
```
Comment (sukritishah15) [inline Python/Sieve_of_Eratosthenes.py:40]: I think from here on everything should go inside a multiline comment.
Also, add time and space complexity.

Thank you!

**Comment by shaswatsingh19** [comment]
done

**Comment by sukritishah15** [comment]
@shaswatsingh19 Why have you added Linear Search?

**Comment by sukritishah15** [comment]
@shaswatsingh19 - If you want to add linear search code, create an issue for it.
Comment - Want to take up in Python. You will be assigned.
PR's are separate for separate issues.

**Pull request merged**